torch/nn/parameter.py

```diff
@@ -46,18 +46,7 @@ def __reduce_ex__(self, proto):
     __torch_function__ = _disabled_torch_function_impl
 
 
-class UninitializedParameter(Parameter):
-    r"""A parameter that is not initialized.
-
-    Unitialized Parameters are a a special case of :class:`torch.nn.Parameter`
-    where the shape of the data is still unknown.
-
-    Unlikely a :class:`torch.nn.Parameter`, uninitialized parameters
-    hold no data and attempting to access some properties, like their shape,
-    will throw a runtime error. The only operations that can be performed on a uninitialized
-    parameter are changing its datatype, moving it to a different device and
-    converting it to a regular :class:`torch.nn.Parameter`.
-    """
+class UninitializedMixin:
```

Comment:
nit: `UninitializedMixin` -> `UninitializedTensorMixin` might be more precise on what it does?